dev/manifest.txt

```diff
@@ -0,0 +1,22 @@
+app
+bin
+config
+conjur
+db
+doc
+lib
+policy
```

Comment:
I *think* we can omit the policy directory. As far as I can tell, it contains Conjur policies that are used in our development environment.

Reply:
yeah, they were moved from the `ci/` dir to `policy/` in [this commit](https://github.com/cyberark/conjur-service-broker/commit/c74087d0760a6895fd98b07f80c0a0417dd41d4d#diff-823412d1eacb67956220e532959f0104603057c88704863ca38e7cd188fda812)

maybe they should be moved to `dev/policy/` now? and not included here

Reply:
Moved `policy` dir to `dev` folder as well 👍